src/main/java/mate/academy/model/ShoppingCart.java

```diff
@@ -0,0 +1,70 @@
+package mate.academy.model;
+
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.MapsId;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "shopping_carts")
+public class ShoppingCart {
+    @Id
+    private Long id;
+    @OneToMany(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
```

Comment:
```suggestion
    @OneToMany
```
lazy fetch type is set by default
and using CascadeType.ALL is not a good practice, it is not flexible if you won't need such behaviour in future
so let's remove cascading